### PR TITLE
Add API

### PR DESCRIPTION
**Pull request by emmanuellekamwa**
In this project, I have:
- Created a new game with the name of your choice by using the API.
- Implemented the "Refresh" button
- Implemented the form "Submit" button
- Used async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,54 @@
...
+    method: 'POST',
+    body: JSON.stringify({ user, score }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((response) => response.json())
+    .then((json) => json);
+};
+const sendData = async () => {
```
Comment (Davidosky007) [inline src/index.js:40]: * I get an error in my console when i hit the submit button and when the page loads this causes the added score not to be updated kindly fix it 🙏 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @emmanuellekamwa 👋, 

Good job so far! 💪 
* Nice looking repo ✔️ 
* Descriptive readme file ✔️ 
* Properly written commit messages ✔️ 
* No linter errors ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.js (hunk 1 above)
Comment (dasileker) [reply src/index.js:40]: - [ ] This issue still persisting, this is related to the webpack => style CSS loader, please fix it.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @emmanuellekamwa,** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
### **Hi @emmanuellekamwa,** 

### **Project Approved ✅**

Approved


Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nidalaa** [comment]
Hello  @emmanuellekamwa  ,

_This message is a result of the second opinion request_

Due to the bug, your request was lost and restored only today. We apologize for the inconvenience.

As I can see - the issue raised by  @Davidosky007  was valid and you managed to fix it.

**Pull request merged**